```
pkg/controller/cronjob/cronjob_controllerv2.go:1: : # k8s.io/kubernetes/pkg/controller/cronjob [k8s.io/kubernetes/pkg/controller/cronjob.test]
pkg/controller/cronjob/cronjob_controllerv2_test.go:1719:49: undefined: fmt (typecheck)
```
/lgtm cancel
Please see the CI failure.

--- a/pkg/controller/cronjob/cronjob_controllerv2_test.go
+++ b/pkg/controller/cronjob/cronjob_controllerv2_test.go
@@ -38,6 +38,7 @@ import (
 	"k8s.io/klog/v2/ktesting"
 	"k8s.io/utils/pointer"
 
+	"fmt"
 	_ "k8s.io/kubernetes/pkg/apis/batch/install"
 	_ "k8s.io/kubernetes/pkg/apis/core/install"
 	"k8s.io/kubernetes/pkg/controller"

